plugins/inventory/azure_rm.py

```diff
@@ -268,8 +269,12 @@ def _enqueue_vmss_list(self, rg=None):
         self._enqueue_get(url=url, api_version=self._compute_api_version, handler=self._on_vmss_page_response)
 
     def _get_hosts(self):
-        for vm_rg in self.get_option('include_vm_resource_groups'):
-            self._enqueue_vm_list(vm_rg)
+        if os.environ.get('ANSIBLE_AZURE_VM_RESOURCE_GROUPS'):
+            for vm_rg in  os.environ['ANSIBLE_AZURE_VM_RESOURCE_GROUPS'].split(","):
```

Comment:
remove one whitespace!
```suggestion
            for vm_rg in os.environ['ANSIBLE_AZURE_VM_RESOURCE_GROUPS'].split(","):
```

Reply:
fixed